packages/web-react/src/components/ScrollView/README.md

```diff
@@ -0,0 +1,48 @@
+# ScrollView
+
+## Usage
+
+### Default (Horizontal)
```

Comment:
That's not true. Use set vertical as default. 
```suggestion
### Default (Vertical)
```

Reply:
fixed